run-integration-tests.py

```diff
@@ -26,35 +26,56 @@
 import tempfile
 
 
-# TODO Scala integration tests
-def run_scala_integration_tests(root_dir, package):
-    pass
+def run_scala_integration_tests(root_dir, version):
+    print("##### Running Scala tests on version %s #####" % str(version))
+    clear_artifact_cache()
+    test_dir = path.join(root_dir, "examples", "scala")
+    test_src_dir = path.join(test_dir, "src", "main", "scala", "example")
+    test_classes = [f.replace(".scala", "") for f in os.listdir(test_src_dir)
+                  if f.endswith(".scala") and not f.startswith("_")]
+    with WorkingDirectory(test_dir):
+        for test_class in test_classes:
+            try:
+                cmd = ["build/sbt", "runMain example.%s" % test_class]
+                print("Running Scala tests in %s\n=====================" % test_class)
+                print("Command: %s" % str(cmd))
+                run_cmd(cmd, stream_output=True, env={"DELTA_VERSION": str(version)})
+            except:
+                print("Failed Scala tests in %s" % (test_class))
```

Comment:
Will this print the original error as well?

Reply:
yes. raise below throws the original exception.